SustainableApplication/tasks/views.py

```diff
@@ -2,13 +2,35 @@
 from .models import Tasks
 from django.http import JsonResponse
 from django.shortcuts import get_object_or_404
+from django.utils.timezone import now
+import random
 
 
 # Create your views here.
```

Comment:
```suggestion
@login_required
```
Should a login required be added here?